packages/angular_devkit/build_angular/src/dev-server/schema.json

```diff
@@ -189,6 +189,16 @@
     "poll": {
       "type": "number",
       "description": "Enable and define the file watching poll time period in milliseconds."
+    },
+    "headers": {
```

Comment:
can you please move this section to the top, maybe under `sslCert` - that way all the serving configuration is together. thanks!

Reply:
Done.